tests/switchmap_/server/api/routes/test_graphql.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+"""Unit tests for the GraphQL API routes in the switchmap-ng application.
+
+This module contains unit tests that verify the functionality and behavior
+of the GraphQL routes for the API.
+"""
+
+import sys
+import os
+
+sys.path.append(
+    os.path.abspath(os.path.join(os.path.dirname(__file__), "../../../../.."))
```

Comment:
1. The script needs to run from either the root directory or the directory in which the file resides.
2. See examples in other test files on how to do this